test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/drools/FireRulesSmokeIntegrationTest.java

```diff
@@ -0,0 +1,115 @@
+
+    @Override
+    protected WorkbenchKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getWorkbenchKieServerScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/rule-project").getFile());
+
+        kieServerControllerClient = KieServerControllerClientProvider.getKieServerControllerClient(deploymentScenario.getWorkbenchDeployment());
+
+        kieServerClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+        kieServerRuleServiceClient = kieServerClient.getServicesClient(RuleServicesClient.class);
+
+        deploymentScenario.getKieServerDeployment().setRouterTimeout(Duration.ofMinutes(5));
+    }
+
+    @Test
+    public void executeSimpleRuleFailoverTest() throws InterruptedException {
+        logger.debug("Register Kie Container to Kie Server");
+        KieServerInfo serverInfo = kieServerClient.getServerInfo().getResult();
+        WorkbenchUtils.saveContainerSpec(kieServerControllerClient, serverInfo.getServerId(), serverInfo.getName(), CONTAINER_ID, CONTAINER_ALIAS, PROJECT_GROUP_ID, RULE_PROJECT_NAME, RULE_PROJECT_VERSION, KieContainerStatus.STARTED);
+        KieServerClientProvider.waitForContainerStart(deploymentScenario.getKieServerDeployment(), CONTAINER_ID);
+        WorkbenchUtils.waitForContainerRegistration(kieServerControllerClient, serverInfo.getServerId(), CONTAINER_ID);
+
+        logger.debug("Set Batch command");
+        List<Command<?>> commands = new ArrayList<Command<?>>();
+        BatchExecutionCommand batchExecutionCommand = commandsFactory.newBatchExecution(commands, KIE_SESSION);
+        commands.add(commandsFactory.newSetGlobal(LIST_NAME, new ArrayList<String>(), LIST_OUTPUT_NAME));
+        commands.add(commandsFactory.newFireAllRules());
+        commands.add(commandsFactory.newGetGlobal(LIST_NAME, LIST_OUTPUT_NAME));
+
+        logger.debug("Fire all rules");
+        ServiceResponse<ExecutionResults> response = kieServerRuleServiceClient.executeCommandsWithResults(CONTAINER_ID, batchExecutionCommand);
+
+        logger.debug("Check result of the drools command");
+        KieServerAssert.assertSuccess(response);
+        ExecutionResults result = response.getResult();
+        assertThat(result).isNotNull();
+        List<String> outcome = (List<String>) result.getValue(LIST_OUTPUT_NAME);
+        assertThat(outcome).hasSize(2);
+
+        assertThat(outcome.get(0)).startsWith(FAST_RULE);
+        assertThat(outcome.get(1)).startsWith(SLOW_RULE);
+
+        System.out.println("***********\n\n" + outcome + "\n\n**********");
```

Comment:
I'd delete this System.out.println. If you need to check its contents, use assert.